templates/galaxy/config/object_store_conf.xml.j2

```diff
@@ -128,49 +133,49 @@ The storage consists of 16 Gen6 type A200 storage nodes and 16 Gen5 type X410 no
             <extra_dir type="temp" path="/data/jwd/object_store_temp/dnb06"/>
             <extra_dir type="job_work" path="/data/jwd/main"/>
         </backend>
-        <backend id="files13" type="disk" weight="0" store_by="uuid">
+        <backend id="files13" type="disk" weight="0" store_by="uuid" device="dnb07">
```

Comment:
Note for a future me, those object stores with the same device, we should remove by updating the database. This means `files13-17` should become in the DB all just `files13`. Then we can remove and clean this file.

Reply:
Does that look ok `UPDATE dataset SET object_store_id = 'files13' WHERE object_store_id = 'files14';`

Reply:
> `UPDATE dataset SET object_store_id = 'files13' WHERE object_store_id = 'files14';`

Query looks fine.